file.go

```diff
@@ -46,41 +52,72 @@ func readFile(file *zip.File) ([]byte, error) {
 	return buff.Bytes(), nil
 }
 
-// Close closes the ExcelFile, rendering it unusable for I/O.
-func (e *ExcelFile) Close() error {
-	return e.zipFile.Close()
+// Close closes the XlsxFile, rendering it unusable for I/O.
+func (e *XlsxFileCloser) Close() error {
```

Comment:
we should probably be a bit more consistent than (e *XlsxFileCloser) - maybe (xl *XlsxFileCloser)